torch/_functorch/_aot_autograd/subclass_parametrization.py

```diff
@@ -79,11 +87,11 @@ def unwrap_tensor_subclass_parameters(module: torch.nn.Module) -> torch.nn.Modul
     becomes: {"parametrizations.p2.original0": torch.Tensor, "parametrizations.p2.original1": torch.Tensor}
 
     """
-    name_param: list[tuple[str, torch.nn.Parameter]] = list(
```

Comment:
fyi @IvanKobzarev (Tugsuu is tweaking the parametrization code here to also apply to subclass buffers)